docs/guides/getting-started/index.html

```diff
@@ -21,7 +21,7 @@ <h2 id="step-1">Step 1: Install HelixUI</h2>
         <hx-tab>unpkg</hx-tab>
       </hx-tablist>
       <hx-tabcontent>
-        <hx-tabpanel class="hxBox-xl">
+        <hx-tabpanel class="hxBox hxXl">
```

Comment:
`hxXl` adds too much space. Please normalize all tabpanels on the page to use `hxLg` instead.